_config.yml

```diff
@@ -3,7 +3,9 @@ description: >
   Berlin Hack and Tell is a monthly meetup where eight people present their hacks. You have five minutes followed by up to five minutes of questions. Show us code, not slides or a sales pitch!
   Follow our [Code of Conduct](https://hackandtell.org/#code_of_conduct) and show off a silly/hobby project that you can't present elsewhere. Think of it like "stand up hacking".
   Check out the past projects below and sign up on our Meetup page!
-url: http://berlinhackandtell.rocks
+#url: http://berlinhackandtell.rocks
+url: https://lucasrangit.github.io
+baseurl: /berlinhackandtell.rocks
```

Comment:
This change is only to support building the static site from my fork. It will not be in the final PR.